airbyte-ci/connectors/pipelines/pipelines/dagger/containers/java.py

```diff
@@ -33,13 +35,15 @@ def with_integration_base_java(context: PipelineContext, build_platform: Platfor
         context.dagger_client.container(platform=build_platform)
         # Use a linux+jdk base image with long-term support, such as amazoncorretto.
         .from_(AMAZONCORRETTO_IMAGE)
+        # Bust the cache on a daily basis to get fresh packages.
+        .with_env_variable("DAILY_CACHEBUSTER", datetime.datetime.now().strftime("%Y-%m-%d"))
         # Install a bunch of packages as early as possible.
         .with_exec(
             sh_dash_c(
                 [
                     # Update first, but in the same .with_exec step as the package installation.
                     # Otherwise, we risk caching stale package URLs.
-                    "yum update -y",
+                    "yum update -y --security",
```

Comment:
Does this update less things (only security) or more things (whatever we updated before AND security)?

Reply:
@evantahler I updates less things (only security):

> The --security parameter is important. Without it, yum update installs all updates, including bug fixes and enhancements. 
([source](https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/managing_and_monitoring_security_updates/installing-security-updates_managing-and-monitoring-security-updates))

Reply:
But don't we want all the updates?

Reply:
I'm not sure, I think I prefer to know that only the security patches might lead to system library version change. The less versions change the less surprises, isn't it?